arangod/IResearch/IResearchViewMeta.cpp

```diff
@@ -593,7 +597,7 @@ bool IResearchViewMeta::init(arangodb::velocypack::Slice const& slice, std::stri
     std::string errorSubField;
 
     auto const field = slice.get(fieldName);
-    mask->_storedValues = field.isArray();
+    mask->_storedValues = !field.isNone();
```

Comment:
why not `field.isArray()`?

Reply:
Because if somebody will supply string or object, this will be silently dropped. And one will not know there is nothing stored. It is more important now, as storedValues are excluded from properties output.